public_html/wp-content/plugins/wordcamp-payments/views/payment-request/metabox-status.php

```diff
@@ -86,23 +96,24 @@ class="large-text"
 	<div id="major-publishing-actions">
 		<?php if ( $current_user_can_edit_request ) : ?>
 
-			<?php if ( !empty( $submit_note ) ) : ?>
+			<?php if ( ! empty( $submit_note ) ) : ?>
 				<div><?php echo $submit_note; ?></div>
 			<?php endif; ?>
 
-
-			<div id="delete-action">
-				<?php if ( current_user_can( 'delete_post', $post->ID ) ) : ?>
+			<?php if ( current_user_can( 'delete_post', $post->ID ) ) : ?>
+				<div id="delete-action">
 					<a class="submitdelete deletion" href="<?php echo get_delete_post_link( $post->ID ); ?>">
 						<?php _e( 'Delete', 'wordcamporg' ); ?>
 					</a>
-				<?php endif; ?>
-			</div>
+				</div>
+			<?php endif; ?>
```

Comment:
I wonder if the `delete-action` element was intentionally left in, even if it's blank, maybe to preserve the layout or something? It might be affected by cases where `#publishing-action` is shown/not-shown.

If it tests well for you though, then it's probably safe to change this. It does make more sense to not create it when the button won't be created, and we can always revert this part if we find a problem in the future.

Reply:
I haven't noticed any issues in testing. I'd prefer to leave it until someone reports a problem.

Reply:
👍